manifests/install.pp

```diff
@@ -17,17 +17,18 @@
   $package_options        = undef,
 ) {
   include tomcat
+  $_catalina_home = pick($catalina_home, $::tomcat::catalina_home, $name)
```

Comment:
Yes, @bmjen, I guess you're right. However, I would've expect that behavior: If I set the default using the class I would want tomcat::install to use that, and not the name from tomcat::install itself, but opinions might differ on this one.

Finally, it's up to you. I don't know a backwards compatible way here, because switching $name and $::tomcat::catalina_home in that statement would also not work, because $name is always set.